madhav/src/main/java/com/lftechnology/phpjava/numbers/AddMultiples.java

```diff
@@ -0,0 +1,26 @@
+package main.java.com.lftechnology.phpjava.numbers;
+
+import main.java.com.lftechnology.phpjava.main.Scanner;
+
+public class AddMultiples {
+    
+    /**
+     * Gets three inputs - limit, number1, and number2 from user.
+     * Then displays sum of all multiples of either number1 or number2 below the limit.
+     */
+    public void getMultiples() {
+        System.out.println("Enter the limit:");
+        int max = Scanner.get().nextInt();
+        System.out.println("Enter the first number:");
+        int num1 = Scanner.get().nextInt();
+        System.out.println("Enter the second number:");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)